src/Domain/Fixture/FixtureCreator.php

```diff
@@ -437,6 +439,66 @@ private function verifySut(): void {
     }
   }
 
+  /**
+   * Composer require all the dev-dependencies of SUT.
+   *
+   * @see https://backlog.acquia.com/browse/ORCA-353
+   *
+   * @throws \Acquia\Orca\Exception\OrcaFileNotFoundException
+   * @throws \Acquia\Orca\Exception\OrcaParseError
+   * @throws \Acquia\Orca\Exception\OrcaException
+   */
+  private function composerRequireSutDevDependencies(): void {
+    $this->output->section('Adding dev-dependencies of SUT');
+    $dev_dependencies = [];
+    $package = $this->options->getSut();
+
+    if ($package === NULL) {
+      $this->output->writeln("No SUT defined");
+      return;
+    }
+    $subextensions = $this->subextensionManager->getByParent($package);
+
+    do {
+      $dev_dependencies[] = $this->computeDevDependenciesByPackage($package);
+      if (count($subextensions) <= 0) {
+        break;
+      }
+      $package = array_shift($subextensions);
+    } while (1);
```

Comment:
Let's use a `foreach` loop here. It's much simpler and easier to read. Just eyeballing it, something like this, I think:

```suggestion
foreach ($packages as $package) {
  $dev_dependencies[] = $this->computeDevDependenciesByPackage($package);
}
```